scripts/build-tdnf-rpms.in

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+## file:    build-tdnf-rpms.sh
+## brief:   Build tdnf rpms using tdnf.spec
+## author:  Shreenidhi Shedi <sshedi@vmware.com>
+
+set -e
+
+arch="$(uname -m)"
+project_name="@PROJECT_NAME@"
+project_dir="@CMAKE_SOURCE_DIR@"
+
+cd "${project_dir}"
+project_version="@PROJECT_VERSION@"
+rpm_build_path="${project_dir}/.rpms"
+full_name="${project_name}-${project_version}"
+
+cleanup()
+{
+  rm -rf "${rpm_build_path}"
+}
+trap cleanup EXIT
+
+echo "Building ${full_name} RPMs"
+git archive --prefix="${full_name}"/ --format=tar.gz -o "${full_name}".tar.gz HEAD
+
+rm -rf "${rpm_build_path}"
+mkdir -p "${rpm_build_path}"/{SOURCES,BUILD,RPMS/"${arch}"}
+
+mv -f "${full_name}".tar.gz "${rpm_build_path}"/SOURCES
+
+rpmbuild --nocheck --nodeps -D "_topdir ${rpm_build_path}" -D "with_check 0" -bb "${project_name}".spec
```

Comment:
Would be better if we add--quiet as well?

Reply:
Also, if you are passing ```with_check 0``` we can remove passing ```--nocheck``` so that if someone wants to run make check should be able to do so.